src/client/gui/lib/catalogue/launch_form.dart

```diff
@@ -317,7 +401,20 @@ class _LaunchFormState extends ConsumerState<LaunchForm> {
     }
 
     initiateLaunchFlow(ref, launchRequest, mountRequests);
-    Scaffold.of(context).closeEndDrawer();
+
+    if (configureNext) {
+      // Reset the form
+      formState.reset();
+      mountFormState?.reset();
+      setState(() {
+        mountRequests.clear();
+        addingMount = false;
+      });
+    }
```

Comment:
Hmm, wasn't the whole point of 'Configure next' to keep your form intact so you can just modify what you want?